packages/office-ui-fabric-react/src/components/OverflowSet/OverflowSet.tsx

```diff
@@ -135,11 +179,90 @@ export class OverflowSet extends BaseComponent<IOverflowSetProps, {}> implements
 
   private _onRenderOverflowButtonWrapper = (items: any[]): JSX.Element => {
     const wrapperDivProps: React.HTMLProps<HTMLDivElement> = { className: css('ms-OverflowSet-overflowButton', styles.item) };
+    const overflowKeytipSequences = this.props.keytipSequences;
+    const itemSubMenuProvider = this.props.itemSubMenuProvider;
+
+    // Register all persisted keytips and save
+    if (overflowKeytipSequences) {
+      items.forEach((overflowItem) => {
+        const keytip = (overflowItem as IOverflowSetItemProps).keytipProps;
+        if (keytip) {
+          // Create persisted keytip
+          const persistedKeytip: IKeytipProps = {
+            content: keytip.content,
+            keySequences: keytip.keySequences,
+            hasDynamicChildren: false,
```

Comment:
Why is this always set to false?

Reply:
I think it can just be omitted, probably leftover from some other code path